anthos-bm-gcp-terraform/resources/run_initialization_checks.sh

```diff
@@ -90,7 +90,7 @@ function __cleanup__ () {
 function __trap_with_arg__ () {
   FUNC="$1" ; shift
   for SIG ; do
-    trap '$FUNC $SIG' '$SIG'
+    trap "$FUNC $SIG" "$SIG"
```

Comment:
Interestingly, the failed test case says otherwise:
```
In ./anthos-bm-gcp-terraform/resources/run_initialization_checks.sh line 93:
    trap "$FUNC $SIG" "$SIG"
          ^---^ SC2064: Use single quotes, otherwise this expands now rather than when signalled.
                ^--^ SC2064: Use single quotes, otherwise this expands now rather than when signalled.
```

Reply:
Ohh probably that's why I had it single quotes in the first place. Let me check

Reply:
I think this still had to be a change done and we might have to skip the linter on this line. The issue is [described here](https://github.com/koalaman/shellcheck/wiki/SC2064). It is applicable when the signal is passed in as is. In our example here we pass in the signals as variables.

So if we use single quotes - allowing on time expansion - will create this error when setting up the trap with a variable. So I have escaped the lint check for this line.

WDYT?

Reply:
So the idea is that you _do_ want the expansion to occur ASAP, or else we run into issues?
Makes sense to me.